### PR TITLE
location updates

### PR DESCRIPTION
**Pull request by drevets**
connects to #5 

map renders initial location of the users + nearby potholes

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -37,7 +38,7 @@ export default class App extends React.Component {
   render() {
     return (
       <Provider store={store}>
-        <BottomLinks />
```
Comment (chawes13) [inline App.js]: Do we not need these navigation components anymore?

File: components/AddPotholeLocation.js [hunk 2]
```diff
@@ -56,8 +56,14 @@ class AddPotholeLocation extends React.Component {
       longitudeDelta: 0.001,
     };
     this.setState({ initialRegion });
+    this._getPotholesAsync(latitude, longitude)
   };
 
+  _getPotholesAsync = async (lat, lon) => {
+    await this.props.getPotholes(lat, lon)
+    this.setState({potholes: this.props.potholes})
```
Comment (chawes13) [inline components/AddPotholeLocation.js]: is it necessary to map these to a local state? Did the reference to this.props.potholes not work in the render?

File: store/potholes.js [hunk 3]
```diff
@@ -1,11 +1,36 @@
...
+    type: GET_POTHOLES,
+    potholes
+  }
+}
+
+//THUNKS
+
+export const fetchPotholes = (lat, lon, latDelt, lonDelt) => {
+  return async dispatch => {
+    const potholes = await axios.get(`http://192.168.1.137:8080/api/potholes/${lat}/${lon}`);
```
Comment (chawes13) [inline store/potholes.js]: can we abstract the IP address into a process.env variable in secrets.js?

Comment (chawes13) [reply store/potholes.js]: We should confirm with chaz  if /lat/lon follows REST or if we should use query params

File: App.js (hunk 1 above)
Comment (drevets) [reply App.js]: added them back in

File: store/potholes.js (hunk 3 above)
Comment (drevets) [reply store/potholes.js]: changed to query params

Comment (drevets) [reply store/potholes.js]: looking into making ip into a variable...necessary for this merge or can do on a later one?

**Pull request merged**